README.md

```diff
@@ -89,8 +89,8 @@ end
 You can also locale other entities through an geolocated association as in:
 ```ruby
 class Job < ActiveRecord::Base
-  belongs_to :job
-  acts_as_geolocated through: :job
+  belongs_to :event
```

Comment:
Did you mean `place` ?

Reply:
I refer to these fixtures
https://github.com/diogob/activerecord-postgres-earthdistance/tree/master/spec/fixtures
but that's true that `belongs_to :place` is better in this README